src/Components/UserFilter/UserFilter.Component.jsx

```diff
@@ -34,8 +34,13 @@ class UserFilter extends Component {
   /**
    * Method to handle closing invite modal
    */
-  handleClose = () => {
-    this.setState({ open: false });
+  handleClose = isSending => {
+    setTimeout(() => {
```

Comment:
Why are we using `setTimeout` here?

Reply:
It allows the form to stay open so isSending can change from false to true , otherwise the modal keeps closing as soon as you hit the submit button